scala3doc/test/dotty/dokka/diagram/HierarchyTest.scala

```diff
@@ -0,0 +1,98 @@
+package dotty.dokka.diagram
+
+import dotty.dokka.ScaladocTest
+import dotty.dokka.Assertion.AfterDocumentablesTransformation
+import dotty.dokka.kUnit
+import dotty.dokka.model.api._
+import scala.jdk.CollectionConverters.{ListHasAsScala, SeqHasAsJava}
+import org.junit.Assert.{assertSame, assertTrue, assertEquals}
+
+class HierarchyTest extends ScaladocTest("hierarchy"):
+    override def assertions = Seq(
+        AfterDocumentablesTransformation { m =>
+            m.visitMembers { x =>
+                if (x.getName == "C1") {
+                    assertEquals(List("A1", "A2[Int]", "A3[Int, String]", "Any", "B1", "B2", "B3", "Object"), x.getParentsAsStrings)
+                    assertEquals(List("B1", "B2", "B3"), x.getDirectParentsAsStrings)
+                    assertEquals(List("E1", "E2"), x.getKnownChildrenAsStrings)
+                    val graph = MemberExtension.getFrom(x).map(_.graph)     
+                    assertEquals(true, graph.isDefined)
```

Comment:
`assertTrue("Graph is empty!", `graph.isDefined`)

Reply:
Will change